examples/demo7.py

```diff
@@ -183,9 +183,9 @@ def main(argv):
     # A Sersic profile may be truncated if desired.
     # The units for this are expected to be arcsec (or specifically -- whatever units
     # you are using for all the size values as defined by the pixel_scale).
-    bulge = galsim.Sersic(half_light_radius = 0.7, n = 3.2, trunc = 8.5, gsparams=gsparams)
-    disk = galsim.Sersic(half_light_radius = 1.2, n = 1.5, gsparams=gsparams)
-    gal5 = 0.4*bulge + 0.6*disk  # Net half-light radius is only approximate for this one.
+    bulge = galsim.Sersic(flux=0.4, half_light_radius=0.7, n=3.2, trunc=8.5, gsparams=gsparams)
+    disk = galsim.Sersic(flux=0.6, half_light_radius=1.2, n=1.5, gsparams=gsparams)
+    gal5 = bulge + disk  # Net half-light radius is only approximate for this one.
```

Comment:
There was a disagreement between demo7.py and demo7.yaml on exactly 1 fft image out of 100, and on only 1 pixel on that image, which was off by exactly 1 ADU.  It turned out that for that pixel the rounding error from rendering the galaxies with flux = 1 and then multiplying by 0.4 and 0.6 was slightly different than rendering directly with flux = 0.4 and 0.6.  Then when the two numbers were used as the mean values for the Poisson deviate noise, they produced values that differed by 1.

I'm not sure precisely which change caused it to break on this branch when it worked on master, but there's probably some slight difference in the order of operations somewhere that changed.  I do at least understand why the old way wasn't necessarily numerically identical between the yaml and py versions.